configure.ac

```diff
@@ -171,8 +171,8 @@ AC_ARG_ENABLE(
 
 AC_ARG_ENABLE(
 	[openssl],
-	[AS_HELP_STRING([--enable-openssl],[enable OpenSSL linkage @<:@detect@:>@])],
-	,
+	[AS_HELP_STRING([--enable-openssl[[=DIR]]],[enable OpenSSL linkage [installed at prefix DIR] @<:@detect@:>@])],
+	[OPENSSL_CFLAGS="-I${enableval}/include" OPENSSL_LIBS="-L${enableval}/lib -lcrypto"],
```

Comment:
OpenSSL may have other dependant libraries...

Reply:
Is there possible assign prefix then to pkg-config search path?
Seems like the "enable-openssl" option is also wrong and should be "with-openssl".

Reply:
I want avoid this
https://github.com/OpenSC/OpenSC/blob/master/MacOSX/build-package.in#L42-L43

Reply:
You may also need to set `PKG_CONFIG_SYSROOT_DIR`

Reply:
Is there reason why we link with libssl?
```
otool -L /usr/local/lib/libopensc.dylib 
/usr/local/lib/libopensc.dylib:
	/usr/local/lib/libopensc.6.dylib (compatibility version 7.0.0, current version 7.0.0)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/local/opt/openssl/lib/libssl.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib (compatibility version 1.0.0, current version 1.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.250.1)
```